api.ts

```diff
@@ -2185,11 +2189,19 @@ export interface SyncTableOptions<
   PermissionsContextT extends SyncPassthroughData,
 > {
   /**
-   * The name of the sync table. This is shown to users in the Coda UI.
+   * The name of the sync table. This is shown to users in the Coda UI if displayName is not present.
    * This should describe the entities being synced. For example, a sync table that syncs products
    * from an e-commerce platform should be called 'Products'. This name must not contain spaces.
+   *
+   * Important: This should not be updated after the first time a table is included in the pack version.
    */
   name: string;
+
+  /**
+   * This is the name shown to users in the Coda UI. If not present, name will be used. This is safe to update.
```

Comment:
Nit: Perhaps use an `{@link}` to link to the name field, for better documentation.